h/api/views.py

```diff
@@ -222,6 +224,9 @@ def _create_annotation(fields, user):
     annotation['user'] = user.id
     annotation['consumer'] = user.consumer.key
 
+    if nipsa.is_nipsad(user.id):
+        annotation["not_in_public_site_areas"] = True
```

Comment:
any reason not to just make this read `"nipsa"`? I think Elasticsearch may compress the keys but when we refer to this as `nipsa` absolutely everywhere else I don't see why we should spell it out here.